glommio/src/lib.rs

```diff
@@ -307,6 +307,7 @@ mod free_list;
 #[allow(dead_code)]
 #[allow(clippy::upper_case_acronyms)]
 mod iou;
+#[macro_use]
```

Comment:
I find it a bit better to put the macros in lib.rs, and avoid the `macro_use`. this is because cargo fmt can reorder imports, and unfortunately macros are sensitive to the order of imports.

Reply:
I moved it lib.rs. But apart from other macro rules to make it is visible to mod `parking` and `sys`.